manifests/init.pp

```diff
@@ -61,7 +61,12 @@
   Optional[Integer] $logconfig_file_max_log_size = undef,
   Optional[Integer] $logconfig_file_backups      = undef,
   Optional[String] $logconfig_file_category      = undef,
-) {
+
+  # Padoptions
+  Etherpad::Stpadoptions $padoptions = {},
+) inherits ::etherpad::params {
```

Comment:
I think it's better to put all parameters in a different class because if you want to change default parameters, you will modify the  `params.pp`  and not the  `init.pp`.

Reply:
In this module we don't use `params.pp` at all and I'm wondering whether we should be introducing it now in a very different way than it's normally used. In the usual pattern you inherit it and they're the defaults but here you only use it inside the class but that doesn't apply here. You could also include it here.

The reason I'm thinking about this is because inheritance in Puppet is usually seen as a bad thing and accepted for default parameters. Hiera data in modules is intended to finally kill the inheritance pattern in the long run. That's why I always like to avoid it.

Reply:
Fixed.